config/default.json.template

```diff
@@ -243,6 +251,18 @@
       "<p>#LINK#</p>",
       "<hr>"
     ],
+    "expired": [
+        "User #UID# has expired, updating account"
+    ],
+    "expired_html": [
+        "<p>User #UID# has expired, updating account</p>"
+    ],
+    "project_expiration": [
+        "Project quota for #NAME# will expire at #DATE#"
```

Comment:
same as below comment (test.json) do not talk about quota

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/aebfa84ebba26c26dc21888b0836aba0b3cd352a